index.d.ts

```diff
@@ -68,6 +70,30 @@ export interface Options<Flags extends AnyFlags> {
 	*/
 	readonly flags?: Flags;
 
+	/**
+  Define subcommands.
+
+  The key is the name of the subcommand and the value is a function that returns an instance of `meow`.
+
+  The following values get passed to the subcommand function:
+  - `helpText`: The help text of parent `meow` instance.
+  - `options`: The options from the parent `meow` instance.
+
+  @example
+  ```
+  commands: {
```

Comment:
The example should be complete and runnable. I think it's better to define the subcommand function at the top-level, for readability.

Reply:
The example should also illustrate how one might check for and call code for the subcommand.